astroplan/tests/test_core.py

```diff
@@ -155,3 +157,326 @@ def test_FixedTarget_ra_dec():
                                                         'SkyCoord')
     assert vega.coord.dec == vega_coords.dec == vega.dec, ('Retrieve Dec from '
                                                            'SkyCoord')
+
+def test_sunrise_sunset_equator():
+    '''
+    Check that time of sunrise/set for an observer on the equator is
+    consistent with PyEphem results (for no atmosphere/pressure=0)
+    '''
+    lat = '00:00:00'
+    lon = '00:00:00'
+    elevation = 0.0 * u.m
+    pressure = 0
+    location = EarthLocation.from_geodetic(lon, lat, elevation)
+    time = Time('2000-01-01 12:00:00')
+    obs = Observer(location=location, pressure=pressure)
+    astroplan_next_sunrise = obs.sunrise(time, which='next').datetime
+    astroplan_next_sunset = obs.sunset(time, which='next').datetime
+
+    astroplan_prev_sunrise = obs.sunrise(time, which='previous').datetime
+    astroplan_prev_sunset = obs.sunset(time, which='previous').datetime
+
+    # Run get_pyephem_sunrise_sunset() to compute analogous
+    # result from PyEphem:
+    pyephem_next_sunrise = datetime.datetime(2000, 1, 2, 6, 3, 39, 150790)
+    pyephem_next_sunset = datetime.datetime(2000, 1, 1, 18, 3, 23, 676686)
+    pyephem_prev_sunrise = datetime.datetime(2000, 1, 1, 6, 3, 10, 720052)
+    pyephem_prev_sunset = datetime.datetime(1999, 12, 31, 18, 2, 55, 100786)
+
+    # Is there an equivalent to assert_allclose() for datetimes?
```

Comment:
I'm comparing with `datetime` objects as outputs because PyEphem outputs datetimes. This way we're comparing apples to apples... and if the `astropy.time` conversion to datetime objects works, shouldn't that be fine?